receiver/dotnetdiagnosticsreceiver/metrics/converter_test.go

```diff
@@ -0,0 +1,81 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.opentelemetry.io/collector/consumer/pdata"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/dotnetdiagnosticsreceiver/dotnet"
+)
+
+func TestMeanMetricToPdata(t *testing.T) {
+	jsonFile := 0
```

Comment:
Is there any significance to these filenames?  It would be preferable for them to denote their content imo.

Reply:
They just contain serialized `Metric`s that were parsed from previous runs of the receiver. Happy to change the naming convention if you want.

Reply:
That'd be my preference but I wouldn't block for it.  Always helps to have the context clearly spelled out for you imo.

Reply:
Good Idea. I changed the files to e.g. `metric.0.json`